server.node.js

```diff
@@ -0,0 +1,251 @@
+//This Code can be run with Reejs@(Node/Bun)
```

Comment:
to serve this file, check https://github.com/esm-dev/esm.sh/blob/e864c7e6bf737f1e2a6499448e190f812bd56bfa/server/server_handler.go#L454

Reply:
haha already found it myself!